wanikani.js

```diff
@@ -1203,14 +1203,22 @@ var wanikani = (function(window, document) {
             } else {
                 var self = this;
                 var success = function(response) {
+                    var requestedInfo = undefined;
```

Comment:
Is `requestedInfo` ever expected to be `undefined`? Perhaps this might work if not:

```js
var requestedInfo = response.requested_information;
requestedInfo = (key_name === "vocabulary") ? requestedInfo.general  : requestedInfo;
```

Good work btw! 🥂

Reply:
Good idea! I made the change, and tested it. Works great.